### PR TITLE
Simplify some strings

### PR DESCRIPTION
**Pull request by brianteeman**
This mainly removes use of the word "whether" and standardises some strings to use "show or hide". It is simpler English and more consistent across all the strings

Note if you grep the language files for the word "whether" there may be some instances still present because those language strings are not used any more in core.

### CODE REVIEW

File: administrator/language/en-GB/en-GB.com_contact.ini [hunk 1]
```diff
@@ -193,7 +193,7 @@ COM_CONTACT_FIELD_PARAMS_TELEPHONE_DESC="Show or hide telephone number."
 COM_CONTACT_FIELD_PARAMS_TELEPHONE_LABEL="Telephone"
 COM_CONTACT_FIELD_PARAMS_TOWN-SUBURB_DESC="Show or hide city or suburb."
 COM_CONTACT_FIELD_PARAMS_TOWN-SUBURB_LABEL="City or Suburb"
-COM_CONTACT_FIELD_PARAMS_VCARD_DESC="Whether or not to allow export to vCard format."
+COM_CONTACT_FIELD_PARAMS_VCARD_DESC="WShow or hide a link to allow export to vCard format."
```
Comment (andrepereiradasilva) [inline administrator/language/en-GB/en-GB.com_contact.ini]: WShow?

File: administrator/language/en-GB/en-GB.ini [hunk 2]
```diff
@@ -403,9 +403,9 @@ JGLOBAL_LINKED_TITLES_LABEL="Linked Titles"
 JGLOBAL_LIST="List"
 JGLOBAL_LIST_ALIAS="(<span>Alias</span>: %s)"
 JGLOBAL_LIST_ALIAS_NOTE="(<span>Alias</span>: %s, <span>Note</span>: %s)"
-JGLOBAL_LIST_AUTHOR_DESC="Whether to show article author in the list of articles."
+JGLOBAL_LIST_AUTHOR_DESC="show or hide the article author in the list of articles."
```
Comment (zero-24) [inline administrator/language/en-GB/en-GB.ini]: Is this expected to be `show` and not `Show`?

File: administrator/language/en-GB/en-GB.com_contact.ini (hunk 1 above)
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_contact.ini]: I said there might be typos ;)

File: administrator/language/en-GB/en-GB.com_content.ini [hunk 3]
```diff
@@ -31,7 +31,7 @@ COM_CONTENT_CONFIGURATION="Articles: Options"
 COM_CONTENT_CREATE_ARTICLE_CATEGORY_LABEL="Default Category"
 COM_CONTENT_CREATE_ARTICLE_CATEGORY_DESC="If set to 'Yes', this page will only let you create articles in the category selected below."
 COM_CONTENT_DRILL_CATEGORIES_LABEL="List or Blog: after choosing the display,<br />make sure you define the Options in the desired layout."
-COM_CONTENT_DRILL_DOWN_LAYOUT_DESC="When drilling down to a category, whether to show articles in a list or blog layout."
+COM_CONTENT_DRILL_DOWN_LAYOUT_DESC="When drilling down to a category show articles in a list or blog layout."
```
Comment (MATsxm) [inline administrator/language/en-GB/en-GB.com_content.ini]: `When drilling down to a category, show articles in a list or blog layout.`
Keeping the comma here?
Must be my franglish, but this one is not "clear" to me.

File: administrator/language/en-GB/en-GB.ini [hunk 4]
```diff
@@ -366,7 +366,7 @@ JGLOBAL_FIELDSET_BASIC="Options"
 JGLOBAL_FILTER_ATTRIBUTES_DESC="3. List additional attributes, separating each attribute name with a space or comma. For example: <i>class,title,id</i>."
 JGLOBAL_FILTER_ATTRIBUTES_LABEL="Filter Attributes<sup>3</sup>"
 JGLOBAL_FILTER_CLIENT="- Select Location -"
-JGLOBAL_FILTER_FIELD_DESC="Whether to show a Filter field for the list. Select Hide to hide the filter field."
+JGLOBAL_FILTER_FIELD_DESC="Show or hide a Filter field for the list."
```
Comment (bertmert) [inline administrator/language/en-GB/en-GB.ini]: "Filter". Why uppercase?

File: administrator/language/en-GB/en-GB.com_content.ini (hunk 3 above)
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_content.ini]: OK will do to keep you happy ;)

File: administrator/language/en-GB/en-GB.ini (hunk 4 above)
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.ini]: It was this way before so I didnt spot this error

File: administrator/language/en-GB/en-GB.ini (hunk 2 above)
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.ini]: Typo - thanks for spotting

**Comment by zero-24** [comment]
I have tested this item :white_check_mark: successfully on 19e40ea859c9de6012f8c40b47200fd88d62fb78<br /><br />looks good now<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11265">issues.joomla.org/joomla-cms/11265</a>.</sub>

**Comment by jeckodevelopment** [comment]
I have tested this item :white_check_mark: successfully on 19e40ea

**Comment by brianteeman** [comment]
RTC - thanks for testing<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11265">issues.joomla.org/joomla-cms/11265</a>.</sub>

**Comment by rdeutz** [comment]
@brianteeman could you have a look at the merge conflicts, thanks.

**Comment by brianteeman** [comment]
@rdeutz yeah I was waiting for all the other PR to be merged first as I would have had to do it again

**Comment by brianteeman** [comment]
I give up - my local git say it is upto date

**Comment by brianteeman** [comment]
@zero-24 can you take a look at this one as well please - one day I will understand git but that day is not today

**Comment by brianteeman** [comment]
@zero-24 thanks for fixing the merge conflicts

@rdeutz this can be merged now - thanks

**Comment by truptikagathara** [comment]
I have tested this item :white_check_mark: successfully on 62b7b90129a57c824358bd411f135712b7466de5<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11265">issues.joomla.org/joomla-cms/11265</a>.</sub>

**Comment by rdeutz** [comment]
Travis fails because a script has moved, I going to merge this anyway, hoping the internet will not implode

**Comment by brianteeman** [comment]
Thanks

**Comment by infograf768** [comment]
`+MOD_LANGUAGES_FIELD_CACHING_DESC="Use the global cache setting the content of this module or disable caching for this module.<br />This should be set to 'No caching' when using Items Associations."`
"setting  the content" does not make sense to me.

Should'nt it be?
+MOD_LANGUAGES_FIELD_CACHING_DESC="Use the global cache setting to cache the content of this module or disable caching for this module.<br />This should be set to 'No caching' when using Items Associations."

**Comment by mbabker** [comment]
Somewhat off topic but why can't this module be cached with associations in use?  That seems like a bug.

**Comment by jeckodevelopment** [comment]
@infograf768 I opened PR #11641 to fix the string.
@mbabker I'm not sure I got your report

**Comment by mbabker** [comment]
The language string JM's pointing out says that `mod_languages` should have caching disabled if item associations are in use.  That doesn't seem right at all.

**Pull request merged**